en/docs/extend/errors/customize-authentication-error-messages.md

```diff
@@ -0,0 +1,28 @@
+# Customize Authentication Error Messages
+
+WSO2 Identity Server has standard error messages for different authentication errors that are encountered. See [Error Codes and
+Descriptions](../../../extend/error-codes-and-descriptions) or more information on the standard error codes and descriptions of
+those errors. There are three types of custom errors handled here:
+
+-   Invalid credentials
+-   Invalid User
+-   Account Lock
+
+!!! note
+    Account Lock errors are returned only when account locking is enabled on the server. Refer [User Account Locking and Account Disabling](../../learn/user-account-locking-and-account-disabling) document to enable account locking.
```

Comment:
Guess we can link this to (../../../guides/identity-lifecycles/account-states-overview/)?